app/sessionStore.js

```diff
@@ -675,6 +675,28 @@ module.exports.runPreMigrations = (data) => {
 }
 
 module.exports.runPostMigrations = (data) => {
+  // fingerprinting protection migration
```

Comment:
We were talking with @bbondy when can we remove some old migrations from session store file and this solution would solve it. What I would suggest as well is to create an upgrade script. So it automatically check if file for that version exists or not. For example: let's say that you are upgrading from 0.10 to 0.15.4. When we start the upgrade process we determinate that we need to check versions 0.10.x to 0.15.4 (it can be many versions in between, hot fixes). Then we check if file `migration/0.10.15.js` (hot fix) exist and if so run both pre and post migration. Then you check if `migration/0.11.js`exist and so on. I would have one file per version and it would be standardized with predefined functions. This way we know exactly what was added/removed in which version.